docs/ci/CI_MONITORING_GUIDE.md

```
@@ -0,0 +1,749 @@
+
+# Count tests per split
+poetry run pytest --collect-only --quiet | wc -l
+```
+
+**Questions to Answer**:
+- How many tests are in the slow split?
+- Are there known slow tests in this group?
+- Is the distribution balanced?
+
+**Expected Output**:
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)